package-lock.json

```diff
@@ -9581,8 +9581,8 @@
       "dev": true
     },
     "moment": {
-      "version": "2.29.3",
-      "resolved": "https://unpm.uberinternal.com/moment/-/moment-2.29.3.tgz",
+      "version": "2.29.4",
+      "resolved": "https://unpm.uberinternal.com/moment/-/moment-2.29.4.tgz",
       "integrity": "sha512-c6YRvhEo//6T2Jz/vVtYzqBzwvPT95JBQ+smCytzf7c50oMZRsR/a4w88aD34I+/QVSfnoAnSBFPJHItlOMJVw=="
```

Comment:
The integrity doesn't seem to be correct as the npm install command should have updated it.

Reply:
I am having an issues with vscode i can't clone github repo to work on it, and i try to do npm install in codespace don't work for me.